### DISCUSSION TITLE
Odblokowywania zaznaczania lub kopiowania

### DISCUSSION THREAD

**Discussion by PolishRoboDog** [FiltersHeroes/PolishAnnoyanceFilters#3143] [Pytania i odp] [answered]
### Moja konfiguracja 
**System operacyjny:** Windows
**Przeglądarka internetowa:** Google Chrome 119.0.6045.159
**Bloker:** AdGuard 4.2.226
**Używane filtry:** 
1. EasyList
2. Oficjalne Polskie Filtry do AdBlocka, uBlocka Origin i AdGuarda
3. Polskie Filtry Elementów Irytujących
### Dodatkowe informacje mogące mieć znaczenie
Czy można przez ten filtr odblokować blokadę zaznaczania i kopiowania artykułów?

Zgłoszenie opublikowane przez użytkownika **zxx**.

**Comment by krystian3w** [comment, marked as answer]
Nie,  porzuciliśmy wsparcie takiego problemu.

Jak odtworzę, to mogę kolejną domenę zgłosić do uAssets. Wtedy wystarczy aktywować uBo Annoyances.

Przy AdGuard dużo zależy czy polska agencja prasowa ma kliknięcia miesięcznie na wysokim poziomie by nie dostać statusu "low visitiors". Jak naprawią to wystarczy pewnie dodać subskrypcję "AdGuard - inne/pozostałe elementy irytujące".
